docs/index.html

```diff
@@ -110,14 +110,14 @@
 <div class="fragment"><div class="line">cart.LineItems.Get(firstProduct, selectedOptions);</div><div class="line">cart.LineItems.Delete(firstProduct, selectedOptions);</div></div><!-- fragment --><h2>Native web view checkout</h2>
```

Comment:
Seems like there are still some places that we missed that still reference a `Native web view checkout`